### ISSUE TITLE
The Location Tracker Tutorial: Part 1 document

### ISSUE THREAD

**Issue by yanzhango** [cloudant-labs/location-tracker-couchapp#6]
The document is not completed. User is hard to follow it. The commands are in the document that do not work.
The information should be added as below:
1) Include Windows OS set up in the document.
2) Include the tools that need to be installed before creating this application. Have the detail links that to download those tools.
3)The prerequisites should be clear and have a detail. Avoid to send a link to IBM developerWorks for end user.

**Comment by ramakrse** [comment]
Agree with Yanzhango.. 

Request to add prerequisites and tools for do this tutorial. Hope forward solution
